test/demo/contextChannels.test.ts

```diff
@@ -0,0 +1,270 @@
+import 'jest';
+import {connect, Fin, Identity, Application} from 'hadouken-js-adapter';
+
+import * as fdc3Remote from './utils/fdc3RemoteExecution';
+
+const testManagerIdentity = {uuid: 'test-app', name: 'test-app'};
+
+const testContext = {type: 'test-context', name: 'contextName1', id: {name: 'contextID1'}};
+
+const startedApps:Application [] = [];
```

Comment:
Is this linted? Whitespace looks a bit off here...

Reply:
Yep, linted. Linter allows zero or one spaces here. But removed space anyway